experiments/marin_models.py

```diff
@@ -0,0 +1,20 @@
+# name for the hf hub. cf llama3_tokenizer
+marin_tokenizer = "stanford-crfm/marin-tokenizer"
+
+# to be clear this is the Olmo 2 template except we use llama3's special tokens
```

Comment:
We're finetuning mostly on OLMo's data, so why shouldn't we be using the olmo template?  Why add llama 3 special tokens, etc.?  Also don't understand the generation tag stuff.

Reply:
The data in Tulu etc. are encoded as chats, so we have free choice over whatever to use here. I thought all else equal it is better to use special formatting tokens (that can be escaped) rather than normal tokens (that cannot be escaped) like Olmo does...

Reply:
added a doc comment